packages/shared/src/components/Feed.spec.tsx

```diff
@@ -336,7 +336,7 @@ it('should open login modal on anonymous upvote', async () => {
   );
   const [el] = await screen.findAllByLabelText('Upvote');
   el.click();
-  expect(showLogin).toBeCalledWith('upvote');
+  expect(showLogin).toBeCalledWith({ trigger: 'upvote' });
```

Comment:
Not urgent, but we can simply use the `AuthTriggers` here right?

Reply:
Done! 

Related but not specific to this: 
In this one I'm always a bit confused with myself, sometimes I read it's better to actually use plain strings in tests then the enums you use in prod, just to make sure that if you change something in enums unintentionally, the tests will catch your error. 
Anyways this is something that we can check as a separate thing on the standards that we use for tests.